### ISSUE TITLE
Two errors when running

### ISSUE THREAD

**Issue by ofadam** [Amethyst-Software/convert-ichat-files#1]
I'm getting the following two errors when running this program on a Mac command line:

Error: Not able to read a 3 byte int.

Failed test on line 167 in /tmp/CiFbuild/Source/ichatReader.c.

**Comment by NikolayBorisov** [comment]
Yep, similar error Failed test on line 221 in /tmp/CiFbuild/Source/ichatReader.c.
